src/Extensions/AzureBlobStorage/AzureBlobEgressProvider.cs

```diff
@@ -49,10 +49,10 @@ public AzureBlobEgressProvider(ILogger logger) : base(logger)
 
                 var bloboptions = new BlockBlobOpenWriteOptions
                 {
-                    BufferSize = BlobStorageBufferSize,
+                    BufferSize = options.CopyBufferSize.GetValueOrDefault(DefaultBlobStorageBufferSize),
```

Comment:
Assuming that `CopyBufferSize` is > 0, we'll use it here...while we don't have the concept of a mandatory 5 Mb minimum like on S3, would it make sense to have a reasonable minimum here? I don't expect that it'll matter much since most people won't be setting this, but I don't know if it's worth protecting people from using really low values (maybe if they mistakenly assume the units are in Mb?)